### PR TITLE
Updated remotecall syntax for v0.5 and Compat and BaseTestNext

### PR DESCRIPTION
**Pull request by Pbellive**
Hi Lars,
             This PR updates MUMPS.jl for julia v0.5. It uses the Compat and BaseTestNext packages to maintain backward compatibility with v0.4. It's been tested on 0.4 and 0.5 but not on 0.3.

Patrick

### CODE REVIEW

File: tests/benchmarkDivGradRealSparseRHS.jl [hunk 1]
```diff
@@ -1,5 +1,10 @@
 using MUMPS
-using Base.Test
+if VERSION >= v"0.5.0-dev+7720"
```
Comment (lruthotto) [inline tests/benchmarkDivGradRealSparseRHS.jl]: wouldn't Compat take care of this?

**Comment by lruthotto** [comment]
Looks all great to me. Just one comment about loading Base.Test. Are you sure we need to use all these if statements?

**Comment by Pbellive** [comment]
Hi Lars,
              We don't need all the if statements. MUMPS.jl doesn't currently use any of the new 0.5 testing functionality that requires BaseTestNext for 0.4 compatibility. However, I think it's a good practice in general to use BaseTestNext in case one wanted to change the testing to use some new features. As far as I understand `Compat` doesn't take care of the functionality of the BaseTestNext if statements. 

Anyway, I can remove the if statements or leave them in. Let me know what you'd prefer.

**Comment by lruthotto** [comment]
Well, alright. Let's leave them in then and see what happens towards 1.0.

**Pull request merged**